testing/run-tests.py

```diff
@@ -0,0 +1,26 @@
+import os
+import subprocess
+
+'''
+Script for running Hibernate Integration tests against the
+Spanner-Hibernate dialect.
+'''
+
+HIBERNATE_TESTS_REPO = 'https://github.com/dzou/hibernate-orm.git'
```

Comment:
Since you're overriding `databases.gradle` anyway, what's preventing you from cloning the original upstream Hibernate repo?

Reply:
Yeah I still have some changes which differ from the upstream:

1. Modify existing gradle file to read from mavenLocal and local Spanner JDBC Driver
2. I include the Spanner JDBC driver in my fork - it's not possible to download this within the script via curl because you have to log in to access the file.

Reply:
By that logic we can also just have the JDBC driver jar in this repo.

Reply:
Ok, that can be done.